cypress/cypress/component/Input/Select.cy.tsx

```diff
@@ -68,3 +68,27 @@ it("is disabled", () => {
 
   cy.get(`select[name=${name}]`).should("be.disabled");
 });
+
+it("select only works", () => {
```

Comment:
Suggestion but not required:

Maybe move it to a new test file called `InputOnly.cy.tsx`

Reply:
moved it and added some more, and sorry for resolving it.. wrong button :D

Reply:
no worries and thanks!